app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProfilesController < ApplicationController
+  def index
+    @original_recipes = current_user.recipes.where(original_recipe_id: nil)
+    @forked_recipes = current_user.recipes.where.not(original_recipe_id: nil)
+  end
+end
```

Comment:
I would strongly suggest keeping it RESTful. The resource is `User` and showing an individual resource is best handled in a `#show` action.